src/test/java/dev/sigstore/fulcio/client/FulcioClientTest.java

```diff
@@ -65,11 +67,60 @@ public void testSigningCert() throws Exception {
           CertificateRequest cReq = new CertificateRequest(keys.getPublic(), signed);
 
           // ask fulcio for a signing cert
-          CertificateResponse cResp = c.SigningCert(cReq, token);
+          SigningCertificate sc = c.SigningCert(cReq, token);
 
           // some pretty basic assertions
-          Assert.assertTrue(cResp.getCertPath().getCertificates().size() > 0);
-          Assert.assertTrue(cResp.getSct().length > 0);
+          Assert.assertTrue(sc.getCertPath().getCertificates().size() > 0);
+          Assert.assertNotNull(sc.getSct());
+        } finally {
+          if (fulcioServer != null) {
+            fulcioServer.shutdown();
+          }
+        }
+      }
+    }
+  }
+
+  @Test
+  public void testSigningCert_NoSct() throws Exception {
+    try (FakeOIDCServer oidcServer = FakeOIDCServer.startNewServer()) {
+      File fulcioConfig = testRoot.newFile("fulcio-config.json");
+      Files.write(oidcServer.getFulcioConfig().getBytes(StandardCharsets.UTF_8), fulcioConfig);
+      // start fulcio client with config from oidc server
+      FulcioWrapper fulcioServer = null;
+      System.out.println("Start new server");
```

Comment:
remind me, is system.out.println typical from a test or is a logger used? I have no idea, I'm just curious and my google-foo is letting me down.

Reply:
oh yeah this is from debugging, I couldn't for the life of me over the last two days get consecutive fulcio instances up while running these tests.